aspnetcore/blazor/call-web-api.md

```diff
@@ -28,9 +28,9 @@ See the following components in the *BlazorWebAssemblySample* sample app:
 
 ## Packages
 
-Reference the *experimental* [Microsoft.AspNetCore.Blazor.HttpClient](https://www.nuget.org/packages/Microsoft.AspNetCore.Blazor.HttpClient/) NuGet package in the project file. `Microsoft.AspNetCore.Blazor.HttpClient` is based on `HttpClient` and [System.Text.Json](https://www.nuget.org/packages/System.Text.Json/).
+Reference the *experimental* [Microsoft.AspNetCore.Components.WebAssembly.HttpClient](https://www.nuget.org/packages/Microsoft.AspNetCore.Components.WebAssembly.HttpClient/) NuGet package in the project file. `Microsoft.AspNetCore.Components.WebAssembly.HttpClient` is based on `HttpClient` and [System.Text.Json](https://www.nuget.org/packages/System.Text.Json/).
```

Comment:
This particular (HttpClient) package didn't get renamed.

Reply:
Thanks ... I'll revert.